wolfcrypt/src/port/atmel/atmel.c

```diff
@@ -896,12 +909,83 @@ int atcatls_verify_signature_cb(WOLFSSL* ssl, const byte* sig, unsigned int sigS
     return ret;
 }
 
+static int atcatls_set_certificates(WOLFSSL_CTX *ctx) {
+    int ret = 0;
+    ATCA_STATUS status;
+  
+    /*Read signer cert*/
+    size_t signerCertSize = 0;
```

Comment:
Please put all variable declarations at top of the function or within a brace section.